src/BookmarkWindow.cpp

```diff
@@ -56,15 +53,15 @@ TBookmarkWindow::TBookmarkWindow(float x, float y, const char *title, const char
 	
 	
 	
-	fSaveAndConnectButton = new BButton(BRect(10, 215, 140, 240), "Connect", B_TRANSLATE("Connect"), new BMessage(CONNECT_CLICKED));
+	fSaveAndConnectButton = new BButton("Connect", B_TRANSLATE("Connect"), new BMessage(CONNECT_CLICKED));
 	fSaveAndConnectButton->SetEnabled(false);
 	
-	fSaveButton = new BButton(BRect(150, 215, 210, 240), "Save", B_TRANSLATE("Save"), new BMessage(SAVE_CLICKED));
+	fSaveButton = new BButton("Save", B_TRANSLATE("Save"), new BMessage(SAVE_CLICKED));
 	fSaveButton->SetEnabled(false);
 	
-	BButton *cancelButton = new BButton(BRect(220, 215, 280, 240), "Cancel", kCancel, new BMessage(B_QUIT_REQUESTED));
+	BButton *cancelButton = new BButton("Cancel", kCancel, new BMessage(B_QUIT_REQUESTED));
 	
-	BLayoutBuilder::Group<>(bgview,B_VERTICAL,2)
+	BLayoutBuilder::Group<>(this,B_VERTICAL,2)
```

Comment:
Add a .SetInsets(B_USE_WINDOW_INSETS) for some padding.
It'd actually look nicer by using a grid layout. Have a look at http://xref.plausible.coop/source/xref/haiku/src/preferences/network/InterfaceAddressView.cpp#103 as an example